crypto-ffi/bindings/swift/WireCoreCrypto/TestHost/ContentView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ContentView.swift
+//  TestHost
+//
+//  Created by Jacob Persson on 10.03.25.
+//
```

Comment:
Is this a useful comment given the existence of `git blame`?

Reply:
Missed this one it's generated by XCode. I'll fix